### PR TITLE
haha

### PR DESCRIPTION
**Pull request by clll7**
23333333333333333333333

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/strongloop/loopback-connector-mongodb"
+    "url": "https://github.com/clarkorz/loopback-connector-mongodb"
   },
   "license": {
     "name": "Dual MIT/StrongLoop",
```
Comment (raymondfeng) [inline package.json]: Please revert the changes to package.json. The version will be updated by our release tools.

**Comment by raymondfeng** [comment]
@clll7 What do you propose to fix? I only see a list of changes from @clarkorz .

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by jannyHou** [comment]
I am closing pr here since it's too old and doesn't have enough context. Please feel free to reopen it if the you need this fix/feature and provide some context please :)
